packages/camera/camera_android_camerax/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## NEXT
+
+* Fix typo in `README.md`.
```

Comment:
Please see the style guide linked from the PR checklist; this needs to be "Fixes".